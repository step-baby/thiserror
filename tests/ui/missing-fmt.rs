use thiserror::EnumDisplay;

#[derive(EnumDisplay, Debug)]
pub enum Error {
    #[display("...")]
    A(usize),
    B(usize),
}

fn main() {}
