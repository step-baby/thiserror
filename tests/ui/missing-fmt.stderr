error: missing #[display("...")] display attribute
 --> tests/ui/missing-fmt.rs:7:5
  |
7 |     B(usize),
  |     ^^^^^^^^
